Homework_7/Task_3.py

```diff
@@ -0,0 +1,6 @@
+numbers = [1, 2, 3, 4, 5, 6, 2, 7, 8, 9, 11]
+unique_numbers = set(numbers)
+if len(unique_numbers) == len(numbers):
+    print(f"This set: {numbers} are unique")
```

Comment:
I am not sure that we need a variable  unique_numbers here
if len(numbers) == len(set(numbers)):
    print(f"This set: {numbers} are unique")